projects/sandbox/datagen/datagen/scripts/timeslides.py

```diff
@@ -305,6 +310,19 @@ def main(
                 future = submit_write(pool, injection_ts, t, **injected_data)
                 futures.append(future)
 
+                # infer redshift based on injected luminosity distance and
+                # cosmology use redshift to calculate source frame masses
+                parameters["redshift"] = cosmo.z_at_value(
```

Comment:
Do we need this? Looks like all the priors specify things in terms of redshift anyway, so `parameters["luminosity_distance"]` is coming from the inverse of this calculation

Reply:
Yeah this should be flipped. I think it's safe to assume that our testing set priors will be defined with redshift as opposed to luminosity distance.